tests/Feature/FlightControllerTest.php

```diff
@@ -2,547 +2,570 @@
 
 namespace Tests\Feature;
 
-use App\Http\Controllers\FlightController;
+use App\Actions\Flight\UpdateFlightAction;
+use App\Http\Controllers\Flight\DeleteFlightController;
+use App\Http\Controllers\Flight\GetFlightController;
+use App\Http\Controllers\Flight\StoreFlightController;
+use App\Http\Controllers\Flight\UpdateFlightController;
 use App\Models\Airline;
 use App\Models\City;
 use App\Models\Flight;
 use Illuminate\Foundation\Testing\RefreshDatabase;
 use Tests\TestCase;
 
-class FlightControllerTest extends TestCase
-{
-    use RefreshDatabase;
-
-    /** @test */
-    public function testGetFlightsForAnAirline()
-    {
-        City::factory(2)->create();
-
-        Airline::factory()->create();
-
-        Flight::create([
-            'airline_id' => 1,
-            'origin_city_id' => 1,
-            'destination_city_id' => 2,
+it('gets flights for an airline ', function () {
+    $origin = City::factory()->create();
+    $destination = City::factory()->create();
+
+    $airline = Airline::factory()->create();
+
+    Flight::create([
+        'airline_id' => 1,
+        'origin_city_id' => 1,
+        'destination_city_id' => 2,
+        'departure_time' => '2023-05-05T14:30:00',
+        'arrival_time' => '2023-05-05T14:31:00',
+    ]);
+
+    $this->json('GET', action(GetFlightController::class))
+        ->assertSuccessful()
+        ->assertJsonFragment([
+            'airline' => $airline->name,
+            'origin' => $origin->name,
+            'destination' => $destination->name,
             'departure_time' => '2023-05-05T14:30:00',
             'arrival_time' => '2023-05-05T14:31:00',
         ]);
-
-        $this->json('GET', action([FlightController::class, 'getFlights']))
-            ->assertSuccessful()
-            ->assertJsonFragment([
-                'airline_id' => 1,
-                'origin_city_id' => 1,
-                'destination_city_id' => 2,
-                'departure_time' => '2023-05-05T14:30:00',
-                'arrival_time' => '2023-05-05T14:31:00',
-            ]);
+});
+
+it('creates a flight with empty parameters', function () {
+    City::factory(2)->create();
+    Airline::factory()->create();
+
+    $emptyAirlineData = [
+        'airline_id' => NULL,
+        'origin_city_id' => 1,
+        'destination_city_id' => 2,
+        'departure_time' => '2023-09-11T11:18',
+        'arrival_time' => '2023-09-11T15:19'
+    ];
+
+    $this->json('POST', action('App\Http\Controllers\Flight\StoreFlightController'), $emptyAirlineData)
```

Comment:
use the class like you did above! or better, change it to the route name as I suggested